### ISSUE TITLE
[Bug]: Failed to query add ons after adding custom collection

### ISSUE THREAD

**Issue by martixy** [mozilla-mobile/fenix#20817]
### Steps to reproduce

1. Enable debug mode
2. Add custom add on collection.
3. Try to configure add-ons or something, add-on page is blank with error "Failed to query Add-ons".

### Expected behaviour

The ability to use and configure the add-ons from the custom collection.

### Actual behaviour

Add-on page is blank with error **"Failed to query Add-ons"**.

### Device name

Nexus 7

### Android version

7.1

### Firefox release type

Firefox Nightly

### Firefox version

93.0a1

### Additional information

> 93.0a1 (Build #2015827505)
AC: 93.0.20210811143137, 5078b5105b
GV: 93.0a1-20210811093702
AS: 82.1.0 

The collection I tried to install, if relevant:  
https://addons.mozilla.org/en-US/firefox/collections/16980816/android-addons/

**Comment by Amejia481** [comment]
Thanks for reporting.
Could you share your a print-screen of your custom Add-on collection?  

![image](https://user-images.githubusercontent.com/773158/129207362-c9d07421-7ec4-40a6-b29a-43fcaa65a5fb.png)

**Comment by martixy** [comment]
After seeing you don't trust me to enter the collection properly, I went back to quadruple-check (I'd already done it several times).

I was going off this page: https://blog.mozilla.org/addons/2020/09/29/expanded-extension-support-in-firefox-for-android-nightly/

1. I noticed the collection id is based on the URL.
2. I noticed that there is a difference between the name in the page and the name in the url. (In the URL the space is replaced by a dash)
3. I thought maybe the name is also based on the URL.
4. I entered the name of the collection with a dash instead of a space.
5. It worked this time.

Admittedly I did miss the tutorial saying not to include spaces in the name.

So I guess I accidentally discovered a workaround for collections with spaces ¯\\\_(ツ)_/¯.
